components/collector/src/source_collectors/api_source_collectors/sonarqube.py

```diff
@@ -27,7 +23,7 @@ async def _get_source_responses(self, *urls: URL) -> SourceResponses:
         response = (await super()._get_source_responses(show_component_url))[0]
         json = await response.json()
         if "errors" in json:
-            raise SonarQubeException(json["errors"][0]["msg"])
+            raise SourceCollectorException(json["errors"][0]["msg"])
```

Comment:
Do the errormessages that we get from SonarQube show that they are SonarQube errors? Otherwise, with this change we lost the information that it was the SonarQube api that threw the error.

Reply:
In the UI, the error is shown with the source so it's always clear which source caused the error. In the logging, the URL is included so there it's clear too.